modules/understanding-the-compliance-scan-status.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * manage-compliance/managing-compliance-20.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="understanding-the-compliance-scan-status_{context}"]
+= Understanding the compliance scan status
```

Comment:
/s Overview of the compliance scan status

WDYT?

Reply:
@kcarmichael08 ^^

Reply:
I think the Style Guides discourage the use of "overview" but we have precedent in the guide already (many section titles with "overview" in them) so I think it would be OK.